lua/bufferline/render.lua

```diff
@@ -235,6 +235,12 @@ local function render(update_names)
 
   -- Create actual tabline string
   local result = ''
+  local offset = tonumber(state.offset)
+  if offset then
+     for i = 1, offset, 1 do
+         result = result..' '
+     end
+  end
```

Comment:
`string.rep(' ', offset)` to return `' '` repeated `offset` times

Reply:
solved in the next commit

(i am new to lua) :D